packages/graphql-codegen-cli/src/presets.ts

```diff
@@ -31,9 +31,9 @@ export async function getPresetByName(
         /** ESM Error code */
         err.code !== 'ERR_MODULE_NOT_FOUND'
       ) {
-        throw new DetailedError(
-          `Unable to load preset matching ${name}`,
-          `
+        throw new Error(
```

Comment:
Does this change play well with listr?

Reply:
Yes, I made this change for Listr support actually.

Listr does not support `DetailledError`, leading to missing information in the console output.